prov/efa/src/efa_cq.c

```diff
@@ -118,16 +117,39 @@ static void efa_cq_read_data_entry(struct efa_wc *wc, int i, void *buf)
 	entry[i].len = (uint64_t)wc->ibv_wc.byte_len;
 }
 
+/* Unlike ibv_poll_cq, wide completion APIs do not write to ibv_wc struct. We need to do this manually.
+ * This is inspired by rdma-core, i.e. `efa_process_cqe` and `efa_process_ex_cqe`.
+ */
+static inline void efa_unsafe_write_ibv_cq_ex_wc(struct ibv_cq_ex *cq, struct ibv_wc *wc) {
+	wc->status = cq->status;
+	wc->vendor_err = ibv_wc_read_vendor_err(cq);
```

Comment:
are these APIs only available for newer rdma-core versions? If so, we should check their availability early in `efa/config.m4`.